From khali@linux-fr.org Mon Jun 12 12:55:28 2006
Date: Mon, 12 Jun 2006 21:55:28 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: LM Sensors <lm-sensors@lm-sensors.org>, Kaiwan N Billimoria <kaiwan@designergraphix.com>
Subject: [PATCH 12/14] hwmon: Fix the Kconfig header
Message-Id: <20060612215528.e72133d5.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-kconfig-header-fix.patch

Hardware monitoring chips don't have to be on the I2C bus.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/hwmon/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/hwmon/Kconfig
+++ gregkh-2.6/drivers/hwmon/Kconfig
@@ -1,5 +1,5 @@
 #
-# I2C Sensor chip drivers configuration
+# Hardware monitoring chip drivers configuration
 #
 
 menu "Hardware Monitoring support"
